libraries/jolpica-schemas/src/jolpica_schemas/data_import.py

```diff
@@ -193,7 +193,7 @@ class TeamDriverImport(F1ImportSchema):
     objects: list[TeamDriverObject] = Field(min_length=1)
 
 
-class RoundForeignKeys(HasSeasonForeignKey, HasCircuitForeignKey):
+class RoundForeignKeys(HasSeasonForeignKey):
```

Comment:
Purposeful change, as this was a mistake. Circuit should be a regular field not a foreign key.